src/controller/productController.js

```diff
@@ -0,0 +1,122 @@
+import jwt from 'jsonwebtoken'; // source to be defined by rene
+import dotenv from 'dotenv';
+import db from '../../database/models/index.js';
+import validateProductInput from '../utils/productValidation.js';
+
+// CONFIG DOTENV
+dotenv.config();
+
+// GETTING THE PRODUCT MODEL
+/* eslint-disable no-console */
+
+class productController {
+  static async addProduct(req, res) {
+    try {
+      // IMPORT MODEL PRODUCT
+      const { product, user } = db;
+
+      // TOKEN TOKEN USING EMAIL as id
+      const token = req.headers.cookie.split('=')[1];
+
+      // const decodedToken = JwtUtility(token);
+
+      const decodedToken = jwt.verify(token, process.env.USER_SECRET);
+      const { id } = decodedToken;
+      const { email } = await user.findOne({ where: { email: id } });
```

Comment:
Given the team is proposing to use IDs instead of emails to sign **JWTs**, you should consider searching the user using ID instead of email on this line.

You will have something like this: `const { email } = await user.findOne({ where: { id } });`

Same applies to all places where you were searching for a user using email attribute.